web/beacon-app/yarn.lock

```diff
@@ -3948,10 +3948,10 @@
     estree-walker "^2.0.2"
     picomatch "^2.3.1"
 
-"@rotational/beacon-core@^2.2.2":
-  version "2.2.2"
-  resolved "https://registry.yarnpkg.com/@rotational/beacon-core/-/beacon-core-2.2.2.tgz#7025642deb6e36d5f091eb1f8e96891199ca7d3e"
-  integrity sha512-esyS0GgeL0fJZC6I15wuFMzV0XD1liqGXk3Rg/U5C0m13OJ9ikgirBpAoKfWhJ+UoocvsPgpjU8j/GFqne/10w==
+"@rotational/beacon-core@^2.3.1":
```

Comment:
I created a new version that should allow us to hide a column  in the table by defining `initialState` props